rpc/write.go

```diff
@@ -4,13 +4,11 @@ import (
 	"context"
 )
 
-type BroadcastedInvokeTransaction interface{}
-
-func (provider *Provider) AddInvokeTransaction(ctx context.Context, broadcastedInvoke BroadcastedInvokeTransaction) (*AddInvokeTransactionResponse, error) {
+func (provider *Provider) AddInvokeTransaction(ctx context.Context, invokeTxn AddInvokeTxnInput) (*AddInvokeTransactionResponse, error) {
```

Comment:
Why use `AddInvokeTxnInput` instead of just `InvokeTxnV1`? (See [here](https://github.com/starkware-libs/starknet-specs/blob/6fadca1f1abf48523c0aa799a10061f40f973468/api/starknet_write_api.json#L19))

Reply:
It's because we used to allow InvokeTxnV0 and InvokeTxnV1 for the rpcv03 spec, which took [broadcastInvokeTxn](https://github.com/starkware-libs/starknet-specs/blob/v0.3.0/api/starknet_api_openrpc.json#L1902)